packages/teams-js/src/internal/mediaUtil.ts

```diff
@@ -188,12 +188,12 @@ export function validateViewImagesInput(uriList: media.ImageUri[]): boolean {
  *
  * @internal
  */
-export function validateScanBarCodeInput(barCodeConfig: media.BarCodeConfig): boolean {
+export function validateScanBarCodeInput(barCodeConfig?: media.BarCodeConfig): boolean {
   if (barCodeConfig) {
     if (
       barCodeConfig.timeOutIntervalInSec === null ||
-      barCodeConfig.timeOutIntervalInSec <= 0 ||
-      barCodeConfig.timeOutIntervalInSec > 60
+      (barCodeConfig.timeOutIntervalInSec != undefined && barCodeConfig.timeOutIntervalInSec <= 0) ||
```

Comment:
>barCodeConfig.timeOutIntervalInSec != undefined [](http://example.com/codeflow?start=7&length=47)

This code was previously assuming `barCodeConfig.timeOutIntervalInSec` was not `undefined` when doing this check. Now it explicitly checks for it.

Note that it is "valid" per the definition of this code for `barCodeConfig.timeOutIntervalInSec` to be `undefined`, it just should not check these timeouts in that case.